ProjectGroupA/src/main/java/com/fundation/search/model/Search.java

```diff
@@ -72,9 +82,18 @@ public Search() {
 		BufferedReader in = new BufferedReader(new InputStreamReader(pDOS.getInputStream()));
 		String inputLine = "";
 		
+		if (!file.getSize().equals("0"))
+			file.sizeToBytes();
+		
 		while ((inputLine = in.readLine()) != null) {
 			if (filterResults(inputLine, file)) {
-				resultFiles.add(new ResultFile(inputLine, file));
+				if (!file.getSize().equals("0")) {
+					if (matchSizeCriteria(inputLine, file)) {
+						resultFiles.add(new ResultFile(inputLine, file));
+					}
```

Comment:
we are using the following format for if else statement: 
if (condition) {
    statements;
} else if (condition) {
    statements;
} else {
    statements;
}

please update to this.

Reply:
done